app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -75,4 +83,5 @@ def delete_in_provider_queue
   COMMON_ATTRIBUTES = {}.freeze
   EXTRA_ATTRIBUTES = {}.freeze
   API_ATTRIBUTES = COMMON_ATTRIBUTES.merge(EXTRA_ATTRIBUTES).freeze
+  # TOWER_KIND = 'ssh'.freeze # default to `ssh` just like Tower does
```

Comment:
Right. I had this line started out for helping the test (as it was originally at the `Ansible::Credential` level. yeah, now removed.